mscore/resourceManager.cpp

```diff
@@ -29,11 +29,23 @@ ResourceManager::ResourceManager(QWidget *parent) :
       {
       setObjectName("ResourceManager");
       setupUi(this);
+      
+#ifndef NDEBUG
+      QPushButton* check_update = new QPushButton();
+      check_update->setText("Check Update");
```

Comment:
Needs to be translatable, so `heck_update->setText(tr("Check Update"));`

Reply:
Thanks. Fixed in https://github.com/musescore/MuseScore/pull/4833/commits/1d26ad066f4fdc3c5897d2e6c467a6de02ce32f4.